src/mcp_server/tools/observability_tools.py

```diff
@@ -181,3 +206,80 @@ def analyze_vllm(
         return _resp(content)
     except Exception as e:
         return _resp(f"Error during analysis: {str(e)}", is_error=True)
+
+def analyze_openshift(
+    metric_category: str,
+    scope: str = "cluster_wide",
+    namespace: Optional[str] = None,
+    time_range: Optional[str] = None,
+    start_datetime: Optional[str] = None,
+    end_datetime: Optional[str] = None,
+    summarize_model_id: Optional[str] = None,
+    api_key: Optional[str] = None,
+) -> List[Dict[str, Any]]:
+    """Analyze OpenShift metrics for a category and scope using the Metrics API logic.
```

Comment:
I don't think you need to mentioned  `Metrics API logic` here in description maybe change to 
`Analyze OpenShift metrics for a given category and scope`

Reply:
Sounds good. Done.